presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaConfig.java

```diff
@@ -45,6 +50,10 @@
     private boolean hideInternalColumns = true;
     private int messagesPerSplit = 100_000;
     private boolean timestampUpperBoundPushDownEnabled;
+    private Optional<String> confluentSchemaRegistryUrl = Optional.empty();
```

Comment:
Can we move the `confluent` related properties to a separate a Config object and introduce a enum to specify the schema-provider type (`LOCAL`, `CONFLUENT` and some new projects). Now we can create independent modules for each of these provider and bind them on demand.

Reply:
Good idea! A string is used Instead of an enum, this way it can be included with the custom table description supplier, as a public static variable rather than updating an enum. What do you think?